qtdialogs/DlgWalletMigration.py

```diff
@@ -618,3 +660,40 @@ def handleUnlockCancelForWatchOnly(self):
          # User does NOT want to create a watching-only wallet: close dialog, do not create wallet
          self.reject()
```

Comment:
This reject does not notify migration end to the backend, but the one at line 520 does. Code at line 520 is repeated in `closeEvent`

You are better off overloading `reject` to and implementing the cleanup there. So long as `migrationComplete` is not True and reject is called, you know you can reply to the backend with `success` set to False.